src/main/scala/org/broadinstitute/dsde/firecloud/model/DUOS.scala

```diff
@@ -93,6 +95,32 @@ object DUOS {
     }
   }
 
+
+  case class StructuredDataRequest(generalResearchUse: Boolean,
+                                   healthMedicalBiomedicalUseOnly: Boolean,
+                                   diseaseUseOnly: Array[Int],
+                                   commercialUseProhibited: Boolean,
+                                   forProfitUseProhibited: Boolean,
+                                   methodsResearchProhibited: Boolean,
+                                   aggregateLevelDataProhibited: Boolean,
+                                   controlsUseProhibited: Boolean,
+                                   genderUseOnly: String,
+                                   pediatricResearchOnly: Boolean,
+                                   IRB: Boolean,
```

Comment:
Nit: I would case this similarly to the other codes and maybe even name it better, `irbRequired` for example.

Reply:
I tend to agree. Worth noting that this affects the request payload so it's part of the service API. As it is now adheres to the tech doc so changing it here would necessitate changing it there too.